analytics/tests/test_activity_views.py

```diff
@@ -99,8 +106,32 @@ def test_activity(self, unused_mock: mock.Mock) -> None:
             result = self.client_get("/activity")
             self.assertEqual(result.status_code, 200)
 
+        # Add data for remote activity page
         RemoteRealmAuditLog.objects.bulk_create([RemoteRealmAuditLog(**data) for data in data_list])
-        with self.assert_database_query_count(6):
+        remote_server = RemoteZulipServer.objects.get(id=1)
+        customer = Customer.objects.create(remote_server=remote_server)
+        plan = CustomerPlan.objects.create(
+            customer=customer,
+            billing_cycle_anchor=timezone_now(),
+            billing_schedule=CustomerPlan.BILLING_SCHEDULE_ANNUAL,
+            tier=CustomerPlan.TIER_SELF_HOSTED_BUSINESS,
+            price_per_license=8000,
+            next_invoice_date=add_months(timezone_now(), 12),
+        )
+        LicenseLedger.objects.create(
+            licenses=10,
+            licenses_at_next_renewal=10,
+            event_time=timezone_now(),
+            is_renewal=True,
+            plan=plan,
+        )
+        RemoteZulipServer.objects.create(
+            uuid=str(uuid.uuid4()),
+            api_key="magic_secret_api_key",
+            hostname="demo.example.com",
+            contact_email="email@example.com",
+        )
```

Comment:
This is fine for now, but we will very likely want to use shared test setup code to create customer objects.